package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "angular-2-local-storage",
-  "version": "3.0.2",
+  "version": "4.0.0",
```

Comment:
Thanks for this heaps! Looks like the version here and in the "changelog" are different? I realise the changelog is a bit of a joke, but that might be more confusing than not having it!

Reply:
Hello, I have fixed the changelog :)